theme/layouts/_header.ejs

```diff
@@ -1,28 +1,21 @@
 <header>
-  <div class="tablet-hide">
-    <div class="col2 header-top">
-      <a href="/" class="cell1 start">
-        <img src="<%= site.root %>/images/logo.png" class="logo" />
-        <p id="title">
-        <%= site.title %>
+  <div>
+    <div id="header-top" class="col2 device">
+      <a href="/" id="header-logo" class="cell1 start">
+        <img src="<%= site.root %>/images/logo.png" alt="Privacy Manager logo" />
+        <p class="tablet-hide">
+          <%= site.title %>
         </p>
       </a>
-      <div class="cell2 end">
-      <% include _language %>
+      <button id="hamburger" class="cell2 end desktop-hide" aria-label="Open primary navigation"></button>
+      <div class="cell2 end tablet-hide">
+        <% include _language %>
       </div>
     </div>
-    <div class="col1">
+    <div class="col1 tablet-hide">
       <% include _navigation %>
```

Comment:
I was a little hesitant to do something like:
```   
<div id="hamburger-menu" class="col1">
  <% include _navigation %>
</div>
```
because it seems like there's no desktop navigation because of id="hamburger-menu". Or we can do something better? what you think?
(`#hamburger-menu` properties are in media query.)

Reply:
Not really sure I understand your concern, also didn't get why you want to declare `hamburger-menu` as a row. What I'm trying to say is that you supposedly can delete whole declaration:
```html
<div id="hamburger-menu" class="desktop-hide">
    <% include _navigation %>
</div>
```

And use only 1 navigation declaration both for Tablet and Desktop.

We can discuss this during the meeting, maybe I failing to explain my point clearly.

Reply:
> 
> We can discuss this during the meeting, maybe I failing to explain my point clearly.

I meant exactly what you wrote in code, so no need for additional discussion I guess!